src/Symfony/Component/Intl/Data/Generator/RegionDataGenerator.php

```diff
@@ -181,7 +181,7 @@ private function generateAlpha2ToAlpha3Mapping(array $countries, ArrayAccessible
 
         foreach ($aliases as $alias => $data) {
             $country = $data['replacement'];
-            if (2 === \strlen($country) && 3 === \strlen($alias) && 'overlong' === $data['reason']) {
+            if (2 === \strlen($country) && 3 === \strlen($alias) && \in_array($data['reason'], ['overlong', 'bibliographic'], true)) {
```

Comment:
seems irrelevant?

Reply:
Probably not , as Alpha2 should be resolved into A variant of Alpha3

Reply:
reverted it.